common/lib/xmodule/xmodule/static_content.py

```diff
@@ -18,28 +18,37 @@
 from path import Path as path
 
 from xmodule.x_module import XModuleDescriptor
+from capa_module import ProblemBlock
+
 
 LOG = logging.getLogger(__name__)
 
 
+# List of XBlocks which use this static content setup.
+# Should only be used for XModules being converted to XBlocks.
+XBLOCK_CLASSES = [
+    ProblemBlock,
+]
```

Comment:
Decided to add support for XBlocks to `static_content.py`. If we need to change the setup, we can do it once all XModules have been converted over in one go instead of doing it for each separately.